services.yaml

```diff
@@ -1519,6 +1707,110 @@ definitions:
   ExcludedPackageName:
     type: string
     example: "com.company.package.bad"
+  ViewBasic:
+    type: "object"
+    properties:
+      id:
+        type: "string"
+        example: P12
+      name:
+        type: "string"
+        example: DB Errors
+      shared:
+        type: boolean
+        example: false
+  ViewFull:
+    type: "object"
+    description: Single view full properties
+    properties:
+      name:
+        type: "string"
+        example: DB Errors
+      description:
+        type: "string"
+        example: This is a DB Errors View
+      shared:
+        type: boolean
+        example: false
+      admin:
+        type: boolean
+        description: TBD - NOT YET IMPLEMENTED
+        example: true
+      default:
+        type: boolean
+        description: TBD - NOT YET IMPLEMENTED
+        example: false
+      filters:
+        type: "array"
+        items:
+          $ref: "#/definitions/Filter"
+      alert_on_new:
+        type: boolean
+        description: TBD - NOT YET IMPLEMENTED
+        example: true
+      alert_on_anomaly:
+        type: boolean
+        description: TBD - NOT YET IMPLEMENTED
+        example: true
+  Folder:
+    description: Single folder properties
+    type: "object"
+    properties:
+      id:
+        type: String
+        example: BDCICD
+      name:
+        type: "string"
+        description: Folder Name
+        example: My Folder
+      shared:
+        type: boolean
+        description: Indicates if this folder visible for all users under this key or only for the folder creator.
```

Comment:
is visible, and i prefer whether over if

Reply:
fixed